docs/operations/clone_api.md

```diff
@@ -98,6 +106,10 @@ used:
 
 Setting label / annotation filters is **optional**. If unset, all labels / annotations will be copied as a default.
 
+#### Template Label & Template Annotation filters
+
+Some network cni such as kube-ovn or ovn-kubernetes will inject network information into the annotations of the VM. It would raise an issue When we want to clone a vm from the target vm. The clone vm would use the same network. So you can use template labels and annotation filters to avoid this problems.
```

Comment:
```suggestion
Some network CNIs such as Kube-OVN or OVN-Kubernetes inject network information into the annotations of a VM. When cloning a VM from a target VM the cloned VM will use the same network. To avoid this you can use template labels and annotation filters.
```

Reply:
yes, Thanks a lot! I will follow your suggestion. Many thanks!